core/frontend/src/tile/Tile.ts

```diff
@@ -400,6 +400,34 @@ export abstract class Tile {
     return false;
   }
 
+  private computePixelSizeScaleFactor(args: TileDrawArgs): number {
+    // Check to see if a model display transform with non-uniform scaling is being used.
+    const tf = args.context.viewport.view.getModelDisplayTransform(args.tree.modelId, Transform.createIdentity());

```

Comment:
Is it necessary to obtain the model display transform vs just pulling the scale out of the TileDrawArgs' transform?

Reply:
I did it this way to just pull out the model display transform rather than looking at the full model to world transform which may have other scaling in it.